src/index.html

```diff
@@ -8,6 +8,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="star active"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="star active"></div>
+      <div class="star active"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
+
+    <div class="stars stars--3">
+      <div class="star active"></div>
+      <div class="star active"></div>
+      <div class="star active"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
+
+    <div class="stars stars--4">
+      <div class="star active"></div>
+      <div class="star active"></div>
+      <div class="star active"></div>
+      <div class="star active"></div>
```

Comment:
nonono, don't use classes to set styles for n-th element, use pseudoclasses instead